app/views/content/landing/return-to-teaching-advisers.md

```diff
@@ -0,0 +1,12 @@
+---
+#title: "Get an adviser"
```

Comment:
I think commenting this out is affecting the tab text @MaxineCl ?

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/016641ee-30d6-420a-b22c-c51924e6fa4b)

Reply:
Just removed it. I must have missed that one when I was putting them all back in.